server/listeners/eventHandlers/probationEvents/offenderManagerChangedEventHandler.ts

```diff
@@ -26,13 +26,25 @@ export default class OffenderManagerChangedEventHandler {
           await this.communityService.assignDeliusRole(comDetails.username.trim().toUpperCase(), COM_ROLE_ID)
         }
 
-        await this.licenceService.updateResponsibleCom(crn, {
-          staffIdentifier: comDetails?.staffIdentifier,
-          staffUsername: comDetails?.username,
-          staffEmail: comDetails?.email,
-          firstName: comDetails?.staff?.forenames,
-          lastName: comDetails?.staff?.surname,
-        })
+        await Promise.all([
+          this.licenceService.updateResponsibleCom(crn, {
+            staffIdentifier: comDetails?.staffIdentifier,
+            staffUsername: comDetails?.username,
+            staffEmail: comDetails?.email,
+            firstName: comDetails?.staff?.forenames,
+            lastName: comDetails?.staff?.surname,
+          }),
+          this.licenceService.updateProbationTeam(crn, {
+            probationAreaCode: newCom.probationArea?.code,
+            probationAreaDescription: newCom.probationArea?.description,
+            probationPduCode: newCom.team?.borough?.code,
+            probationPduDescription: newCom.team?.borough?.description,
+            probationLauCode: newCom.team?.district?.code,
+            probationLauDescription: newCom.team?.district?.description,
+            probationTeamCode: newCom.team?.code,
+            probationTeamDescription: newCom.team?.description,
+          }),
+        ])
```

Comment:
These are very likely to update the same row, so just a thought about concurrent update? It's only a thought - as I'm certain that JPA will deal with it as separate transactions and all will be fine - just a thought. Two sequential awaits would take the question away, but I'm 97% certain it'll be fine as is.